kinley-matt/styles/base.css

```diff
@@ -0,0 +1,44 @@
+body {
+  line-height: 1.5;
+  font-size: 100%;
+  color: #313131;
+  box-sizing: border-box;
+}
+
+hr {
+  margin: 1em;
+  width: 80%;
+}
+
+/*TEXT STYLES
+=======================================*/
+p {
+  font-size: 1em;
+}
+
+address {
+  display: inline-block;
+}
+
+/*LINK STYLES
+=======================================*/
+
+a, a:link, a:active {
+  text-decoration: none;
+  color: #777;
+}
+
+a:hover, a:focus {
+  color: #29506D;
+}
+
+a:visited {
+}
+
+/*IMAGE STYLES
+=====================================*/
+
+img {
+  max-width: 100%;
+  width: 20vw;
```

Comment:
It is redundant to have both `max-width` and `width` if both are in responsive units. If you want to use both, `width` should be in pixels or ems e.g.